main.js

```diff
@@ -65,7 +65,7 @@ var obstacleSaw = function(x, y) {
 };
 
 var obstacleRect = function(y) {
-  （500，500，100，25)
+  ctx.rect(500, 500, 100, 25);
```

Comment:
All that was missing was the `ctx.rect` function call. Remember that in real JavaScript we need to use `ctx.rect()` instead of `rect()` as in Khan Academy.